chingsley/ireporter
# Ft fetch addmedia #163061729

#### What does this PR do?
This pull request deals with implementation of fetch to add images and videos to a record (red-flag or intervention).

#### Description of Task to be completed?
- create the editrecord.js file to implement the fetch
- using fetch consume the API with the route /api/v1/<record-type>/addImage  and /api/v1/<record-type>/addVideo with separate fetch calls, setting up the request object to use the 'PATCH' method and the form data to contain the the files to be uploaded.
- Display error messages by calling the 'showDialogMsg()' function with the right arguments
- on successful fetch, display a success message and reload the page so that users can instantly see the uploaded media.

#### How should this be manually tested?
- clone the repo
- cd into the root directory
- run `npm install' to install the necessary dependencies
- run `npm test' to see the tests pass
- run `npm start' to start the server
- ensure that you are a registered user and that you have created some records already. To add image/images, click on the 'choose a file' input element in the images section, if the file chosen is valid, this action will enable the upload button. Click on the upload button to upload the selected image/images. Repeat similar steps in the videos section to upload video/videos. 

#### What are the relevant pivotal tracker stories
#163061729
